Cargo.toml

```diff
@@ -48,6 +50,9 @@ pqcrypto-traits = { version = "0.3", optional = true }
 
 # ring crypto proivder
 ring = { version = "^0.16.2", optional = true, features = ["std"] }
+# libsodium crypto provider
+sodiumoxide = { version = "0.2.5", optional = true }
+byteorder = { version = "1.3.2", optional = true }
```

Comment:
Unless necessary for compatibility or security, can we set these to just "0.2" and "1.3" to give users of the library a bit more flexibility to reduce build times and dependencies?